Source/BusinessMomentsAdaptorImage/Dockerfile

```diff
@@ -1,24 +1,44 @@
-# add stuff here
-FROM node:14.2
+FROM node:14.2.0 AS shared-build
 
-# Create app directory
 WORKDIR /app
 
-# Install app dependencies
-# A wildcard is used to ensure both package.json AND package-lock.json are copied
-# where available (npm@5+)
 COPY package.json ./
+COPY .eslintrc.js ./
+COPY tsconfig.json ./
+#COPY application.json ./
 
-RUN npm install -g typescript ts-node
-#RUN yarn add typescript
-#RUN yarn add ts-node
+COPY Source/Shared/Backend/package.json ./Source/Shared/Backend/package.json
```

Comment:
What is the purpose of copying these config files?

Reply:
@parhambarazesh to get some of the standard packages we use.

Reply:
I have seen a similar structure in the other dockerfiles. That looks meaningful to me. Though I'm not sure how we are going to use these packages. As far as I know, they are not used within the same microservice. Maybe we need them elsewhere?